### ISSUE TITLE
Native features copy/paste is not working in the editor

### ISSUE THREAD

**Issue by loicgeek** [glynskyi/zefyrka#8]

**Comment by loicgeek** [comment]
@loicgeek the copy/paste in the ZefyrEditor

**Comment by thanhhai08sk** [comment]
Happening to me too, on both mobile and web.
```
[✓] Flutter (Channel stable, 2.2.1, on macOS 11.3 20E232 darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.2)
[✓] IntelliJ IDEA Community Edition (version 2021.1.1)
[✓] VS Code (version 1.42.1)
[✓] Connected device (3 available)

• No issues found!

```

**Comment by glynskyi** [comment]
fixed in [zefyrka: ^1.0.4](https://pub.dev/packages/zefyrka/changelog#104)
